packages/main/src/Toast.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="ui5-toast-root-parent"
+     dir="{{rtl}}">
+	<div class="ui5-toast-root"
+	     placement="{{placement}}"
```

Comment:
Better remove this, because it`s strange when the user did not set placement, but we will still render the placement attribute with "BottomCenter" (default one). I suppose you need this to match the following style.

```css
:host([placement="BottomCenter"]) .ui5-toast-root-parent {
	justify-content: center;
	align-items: flex-end;
}
```
As `BottomCenter` is the default placement, you can set these styles
to the `.ui5-toast-root-parent` unconditionally/always and you will cover the case when no placement is given by the user.